examples/image_classification/requirements.txt

```diff
@@ -0,0 +1,7 @@
+torch==1.13.1 
```

Comment:
Since torch will be automatically installed with update-moreh cmd, this looks unnecessary.

Reply:
update-moreh will install torch 1.7.1, but torch==1.13.1 is required for peft